docs/source/conf.py

```diff
@@ -19,17 +17,17 @@ def setup(app):
 
 # Modules that automodapi will document need to be available
 # in the path:
-sys.path.insert(0, str(REPO_ROOT/"src"/"stpipe"))
+sys.path.insert(0, str(REPO_ROOT / "src" / "stpipe"))
 
-# Read the package's setup.cfg so that we can use relevant
+# Read the package's `pyproject.toml` so that we can use relevant
 # values here:
-conf = ConfigParser()
-conf.read(REPO_ROOT/"setup.cfg")
-setup_metadata = dict(conf.items("metadata"))
+with open(REPO_ROOT / "pyproject.toml", "rb") as configuration_file:
+    conf = tomli.load(configuration_file)
+setup_cfg = conf['project']
 
 project = setup_metadata["name"]
```

Comment:
```suggestion
project = setup_cfg["name"]
```

This should fix the failing Docs job. A bigger question is should we just create a RTD and have RTD build the docs?